src/chapters/angular-wk1/file-structure.rst

```diff
@@ -0,0 +1,73 @@
+Angular File Structure
+=======================
+
+Each Angular project contains a standard file structure, which is shown in the
+figure below.
+
+.. figure:: ./figures/AngularFileStructure.png
+   :alt: Visual of the standard Angular file structure.
+
+All of the project files (and there will be LOTS) are stored in the single,
+top-level folder, ``projectName``, which contains a ``src`` folder and a set of
+support files.
+
+.. admonition:: Note
+
+   When you start a new Angular project, do not worry about the support files.
+   These will be generated automatically, and they take care of the routine
+   technical details for making your project run smoothly.
+
+``src`` contains the ``app`` folder and four important files: index.html,
+main.ts, style.css, and polyfills.ts. We will explore these files in more
+detail later [TODO: Rephrase].
```

Comment:
Maybe break this into 2 sections. Entire webpage metadata (main.ts, style.css, polyfills.ts, index.html), and our first component -- the app component. Inside the app folder the files that are used only by the app component. When we add new components that will be used by the app component, we will add them inside the app folder.

Reply:
The students do not know the term "metadata", so mentioning it here will ironically impart no information. I understand where you want to go with this, but I'm not sure that would be a good idea, especially given how quickly we are skimming through Angular.  Let's touch base on this.

Reply:
We will have to rewrite this section because of using Angular CLI and the terminal. So we are not going to address this now. Resolved.